Fix build system to set hypre libraries properly.

Index: petsc-3.4.2/config/PETSc/packages/hypre.py
===================================================================
--- petsc-3.4.2.orig/config/PETSc/packages/hypre.py	2013-08-20 15:33:12.000000000 +0200
+++ petsc-3.4.2/config/PETSc/packages/hypre.py	2013-08-20 15:33:12.000000000 +0200
@@ -7,7 +7,7 @@
                          'http://ftp.mcs.anl.gov/pub/petsc/externalpackages/hypre-2.9.1a.tar.gz']
     self.functions = ['HYPRE_IJMatrixCreate']
     self.includes  = ['HYPRE.h']
-    self.liblist   = [['libHYPRE.a']]
+    self.liblist   = [['libHYPRE_utilities.a', 'libHYPRE_struct_mv.a', 'libHYPRE_struct_ls.a', 'libHYPRE_sstruct_mv.a', 'libHYPRE_sstruct_ls.a', 'libHYPRE_IJ_mv.a', 'libHYPRE_parcsr_ls.a']]
     self.license   = 'https://computation.llnl.gov/casc/linear_solvers/sls_hypre.html'
     # Per hypre users guide section 7.5 - install manually on windows for MS compilers.
     self.worksonWindows   = 1
