Change MUMPS graph partition dependency from ParMETIS to Scotch.

Index: petsc/config/PETSc/packages/MUMPS.py
===================================================================
--- petsc.orig/config/PETSc/packages/MUMPS.py
+++ petsc/config/PETSc/packages/MUMPS.py
@@ -19,8 +19,8 @@
     self.blasLapack = framework.require('config.packages.BlasLapack',self)
     self.blacs      = framework.require('PETSc.packages.blacs',self)
     self.scalapack  = framework.require('PETSc.packages.SCALAPACK',self)
-    self.parmetis   = framework.require('PETSc.packages.ParMetis',self)
-    self.deps       = [self.parmetis,self.scalapack,self.blacs,self.mpi,self.blasLapack]
+    self.scotch     = framework.require('PETSc.packages.Scotch',self)
+    self.deps       = [self.scotch,self.scalapack,self.blacs,self.mpi,self.blasLapack]
     if self.framework.argDB.get('download-scotch') or self.framework.argDB.get('with-scotch'):
       self.scotch     = framework.require('PETSc.packages.PTScotch',self)
       self.deps.append(self.scotch)
