Small adaptations to use Debian sprng.

Index: petsc/config/PETSc/packages/sprng.py
===================================================================
--- petsc.orig/config/PETSc/packages/sprng.py
+++ petsc/config/PETSc/packages/sprng.py
@@ -6,7 +6,7 @@
     self.download  = ['http://ftp.mcs.anl.gov/pub/petsc/externalpackages/sprng-1.0.tar.gz']
     self.functions = ['make_new_seed_mpi'] 
     self.includes  = ['sprng.h'] 
-    self.liblist   = [['liblcg.a']]
+    self.liblist   = [['libsprng.a']]
     return
 
   def setupDependencies(self, framework):
Index: petsc/src/sys/random/impls/sprng/sprng.c
===================================================================
--- petsc.orig/src/sys/random/impls/sprng/sprng.c
+++ petsc/src/sys/random/impls/sprng/sprng.c
@@ -15,7 +15,7 @@
 PetscErrorCode  PetscRandomSeed_Sprng(PetscRandom r)
 {
   PetscFunctionBegin;
-  init_sprng(r->seed,SPRNG_DEFAULT);
+  init_sprng(SPRNG_CMRG,r->seed,SPRNG_DEFAULT);
   PetscFunctionReturn(0);
 }
 
