Small adaptations to use Debian sprng.

Index: petsc-3.3.dfsg/config/PETSc/packages/sprng.py
===================================================================
--- petsc-3.3.dfsg.orig/config/PETSc/packages/sprng.py	2012-09-12 08:26:27.000000000 +0200
+++ petsc-3.3.dfsg/config/PETSc/packages/sprng.py	2012-09-12 08:59:02.000000000 +0200
@@ -6,7 +6,7 @@
     self.download  = ['http://ftp.mcs.anl.gov/pub/petsc/externalpackages/sprng-1.0.tar.gz']
     self.functions = ['make_new_seed_mpi'] 
     self.includes  = ['sprng.h'] 
-    self.liblist   = [['liblcg.a']]
+    self.liblist   = [['libsprng.a']]
     return
 
   def setupDependencies(self, framework):
Index: petsc-3.3.dfsg/src/sys/random/impls/sprng/sprng.c
===================================================================
--- petsc-3.3.dfsg.orig/src/sys/random/impls/sprng/sprng.c	2012-09-12 08:26:27.000000000 +0200
+++ petsc-3.3.dfsg/src/sys/random/impls/sprng/sprng.c	2012-09-12 08:59:02.000000000 +0200
@@ -15,7 +15,7 @@
 PetscErrorCode  PetscRandomSeed_Sprng(PetscRandom r)
 {
   PetscFunctionBegin;
-  init_sprng(r->seed,SPRNG_DEFAULT);
+  init_sprng(SPRNG_CMRG,r->seed,SPRNG_DEFAULT);
   PetscFunctionReturn(0);
 }
 
